packages/playwright-core/src/client/browserContext.ts

```diff
@@ -269,6 +269,13 @@ export class BrowserContext extends ChannelOwner<channels.BrowserContextChannel>
     await this._channel.clearCookies();
   }
 
+  async removeCookies(cookieNames: string | string[]): Promise<void> {
```

Comment:
The functionality you are implementing is more of a "remove cookies by name". I wonder if you'd like to be a bit more proactive and include optional `domain` and `path` parameters. 

```js
async removeCookies(cookies: { name?: string, domain?: string, path?: string });
```

that way you could be able to clear all cookies by name, all cookies for a given domain, all cookies for a given path and their combinations:

```js
await context.removeCookies({ name: 'session-id' });
await context.removeCookies({ domain: 'my-origin.com' });
await context.removeCookies({ path: '/api/v1' });
await context.removeCookies({ name: 'session-id', domain: 'my-origin.com' });
```

Reply:
Thank you for comments! Fixed. Please let me know if there is something else I could change or fix.